src/disco/containers/InstallButton.js

```diff
@@ -143,9 +152,11 @@ export function mapDispatchToProps(dispatch) {
       });
     },
 
-    uninstall({ guid, installURL, slug }) {
+    uninstall({ guid, installURL, slug, name, type }) {
       const addonManager = new AddonManager(guid, installURL);
       dispatch({type: 'START_UNINSTALL', payload: {slug}});
+      const action = type === THEME_TYPE ? 'theme' : 'addon';
+      tracking.sendEvent({action, category: UNINSTALL_CATEGORY, label: name});
```

Comment:
I'm guessing if we change the new AMO we'd want the disco pane to use the new format as well. Either way though.

I'm mostly just not crazy about the `const action = type === THEME_TYPE ? 'theme' : 'addon'` bit. If it used the `addonTypeToAction` from above that would make me happier with it. This isn't likely to change but still feels like something that should be wrapped up in the tracking library to me.